sdk/python/kfp/cli/diagnose_me/dev_env.py

```diff
@@ -0,0 +1,64 @@
+# Lint as: python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License,Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions for diagnostic data collection from development development."""
+
+import enum
+from typing import Optional
+from . import utility
+
+
+class Commands(enum.Enum):
+  """Enum for gcloud and gsutil commands."""
+  PIP3LIST = 1
+  PYTHON3PIPLIST = 2
+  PIP3VERSION = 3
+  PYHYON3PIPVERSION = 4
+
+
+_command_string = {
```

Comment:
Maybe we should report the paths of both (pip3 and python3) as part of diagnostics.

Reply:
Agreed :) updated accordingly.